geoutils/georaster.py

```diff
@@ -15,6 +15,8 @@
 from rasterio.warp import Resampling
 from rasterio.plot import show as rshow
 import matplotlib
+import pyproj
+from scipy.ndimage import map_coordinates
```

Comment:
`scipy` is not in the `setup.py` requirements.

Reply:
Done